src/discussions/posts/NoResults.jsx

```diff
@@ -0,0 +1,31 @@
+import { useSelector } from 'react-redux';
+
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import { selectAreThreadsFiltered } from '../data/selectors';
+import messages from '../messages';
+
+function NoResults({ intl }) {
```

Comment:
This doesn't seem to be used?

Reply:
It got removed as part of resolving merging in master.

When filtering the Post views (not Topics) it should show "No Results found" if there isn't anything to show. Re-added.